package/windows/agent.ps1

```diff
@@ -0,0 +1,2 @@
+Start-Process -NoNewWindow -Wait -FilePath "c:\etc\rancher\agent.exe" -ArgumentList $args
```

Comment:
Do we need to worry about prefixPath here?

Reply:
this is inside the container, it will always be in c:\etc\rancher